magstock/__init__.py

```diff
@@ -57,7 +65,7 @@ def food_cost(self):
 
     @property
     def auto_food(self):
-        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE]
+        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE] or hasattr(self, "auto_food_extra") and self.auto_food_extra
```

Comment:
Rather than do the `auto_food_extra` attribute, why not just do the `self.group and self.group.band is not None` check in here?